brightray/browser/browser_main_parts.cc

```diff
@@ -168,7 +168,9 @@ BrowserMainParts::~BrowserMainParts() {
 void OverrideAppLogsPath() {
 #if defined(OS_WIN)
   std::wstring app_name = base::UTF8ToWide(GetApplicationName());
-  std::wstring log_path = L"%HOMEDRIVE%%HOMEPATH%\\AppData\\Roaming\\";
+  std::wstring drive = _wgetenv(L"HOMEDRIVE"));
+  std::wstring path = _wgetenv(L"HOMEPATH"));
+  std::wstring log_path = drive + "\\" + path + L"\\AppData\\Roaming\\";
```

Comment:
Maybe we could do this instead of both `log_path` and `app_log_path`:

```
std::wstring app_log_path = drive + "\\" + path + L"\\AppData\\Roaming\\" + app_name + L"\\logs";
```